governance/vote-proxy-setup-airgapped-machine/vote-proxy-setup-airgapped-machine.md

```diff
@@ -5,7 +5,7 @@ Level: Intermediate
 Estimated Time: 60 minutes
 
 ## Overview
-This guide describes one way to link an offline cold wallet to an online hot wallet for participating in MakerDAO governance by voting using the [Voting-Proxy contract](https://github.com/makerdao/vote-proxy) as described in this [article](https://medium.com/makerdao/the-makerdao-voting-proxy-contract-5765dd5946b4)
+This guide describes one way to link an offline cold wallet to an online hot wallet for participating in MakerDAO governance by voting using the [Voting-Proxy contract](https://github.com/makerdao/vote-proxy) as described in this [article](https://blog.makerdao.com/the-makerdao-voting-proxy-contract/)
```

Comment:
I tried to go through this guide as well and got stuck at publishing my transaction:
```
Error! Unable To Broadcast Txn : {"jsonrpc":"2.0","id":1,"error":{"code":-32000,"message":"insufficient funds for gas * price + value"}}
```

Maybe I missed something, but got frustrated. 
Will try again another day.